code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { Link } from 'react-router-dom'
+
+import { API_KEY } from './MovieList'
+import { MovieBio } from '../components/MovieBio'
+import { ReactComponent as Arrow } from '../components/arrow.svg';
+import './moviedetails.css'
+
+
+export const MovieDetails = () => {
+  const { id } = useParams()
+  const [movie, setMovie] = useState ([])
+  const URL_MOVIE_DETAILS = `https://api.themoviedb.org/3/movie/${id}?api_key=${API_KEY}&language=en-US`;
+
+  const fetchDetails = () => {
+    fetch(URL_MOVIE_DETAILS)
+      .then((response) => response.json())
+      .then((json) => setMovie(json));
+  };
+  useEffect(fetchDetails, [id]);
+
+  return (
+    
+    <section className="detail-section">
+      <Link to="/" exact="true">
```

Comment:
There is limited set of props available for `<Link />` predefined component provided by `react-router-dom` package and `exact` is not one of them - basically, it's not doing anything as it's not available for it. It's only available for `<Route />` and `<NavLink />` components.

For further reference check out [https://reactrouter.com/web/api/Link](https://reactrouter.com/web/api/Link)